css/style.css

```diff
@@ -346,6 +345,7 @@ video {
   margin: 10px;
   list-style-position: initial;
```

Comment:
The `list-style-` styles and `border-radius` are not doing anything here, please delete them. Also, instead of overwrite the margin-bottom, you could use the short hand in margin on line 346 like this: `margin: 10px 10px 50px`